### DISCUSSION TITLE
Error updating fates via ./bin/git-fleximod

### DISCUSSION THREAD

**Discussion by mpaiao** [NGEET/fates#1292] [General]
I just updated CTSM on my local computer, and I tried to sync all the modules, but I am getting the following error when it tries to update FATES using the newer `./bin/git-fleximod` approach:

```
git_fleximod.git_fleximod - INFO - ls-remote
git_fleximod.git_fleximod - INFO - https://github.com/mpaiao/fates.git
git_fleximod.git_fleximod - INFO - remote
git_fleximod.git_fleximod - INFO - origin	https://github.com/mpaiao/fates.git (fetch)
origin	https://github.com/mpaiao/fates.git (push)
upstream	https://github.com/NGEET/fates.git (fetch)
upstream	https://github.com/NGEET/fates.git (push)
git_fleximod.git_fleximod - INFO - fetch
Cmd('git') failed due to: exit code(128)
  cmdline: git fetch newremote.00
  stderr: 'fatal: 'newremote.00' does not appear to be a git repository
fatal: Could not read from remote repository.
```

Has anyone seen anything similar to this error, or has any suggestion on how to fix this? Thanks!

**Comment by glemieux** [comment]
Was this your first time using `git-fleximod`?  I seem to recall that I had a similar error when I first started using it that requires the upstream remote to be `origin`.

**Comment by mpaiao** [comment]
Yes, first time, I had updated the last time just before they switched to `git-fleximod`. 
To confirm, do you mean that the "origin" remote branch should be set to `https://github.com/NGEET/fates.git`?

**Reply by glemieux** [reply]
Yep, that's what I found.

**Comment by mpaiao** [comment]
I tried swapping "origin" to point to https://github.com/NGEET/fates.git and it did not work. In the end, I deleted my local CTSM git repository entirely, and cloned it again, then it worked. I added a remote repository to my own fates fork (named it something else), and it seems to be working fine now. I am adding this in case someone else gets a similar error.

**Comment by mpaiao** [comment]
Update: it seems that the problem is that git-fleximode does not accept when two remotes are defined in the FATES git, at least in my machine. Has anyone else experienced similar issues?
